### DISCUSSION TITLE
Texture Interpolation

### DISCUSSION THREAD

**Discussion by schwarzsky** [donmccurdy/glTF-Transform#688] [Q&A] [answered]
Is there a way to make texture interpolation to be `Closest` -like in Blenders- with library?

**Comment by donmccurdy** [comment, marked as answer]
Hi @schwarzsky! Yes, texture settings are given on the [`TextureInfo`](https://gltf-transform.donmccurdy.com/classes/core.textureinfo.html) class (a union of the glTF specification's "TextureInfo" and "Sampler" definitions). To change the interpolation for a single use of a single texture:

```js
import { TextureInfo } from '@gltf-transform/core';

const material = document.getRoot().listMaterial()
  .find((mat) => mat.getName() === 'MyMaterial');

const info = material.getBaseColorTextureInfo();
info.setMinFilter(TextureInfo.MinFilter.NEAREST);
info.setMagFilter(TextureInfo.MagFilter.NEAREST);
```

**Reply by schwarzsky** [reply]
Thanks for reply. Great work!
